### PR TITLE
Dashboard documentation updates

### LINKED ISSUE #2197: documentation about fullscreen/auto refresh

### PR DESCRIPTION
**Pull request by kdoh**
closes #2197

### CODE REVIEW

**Comment by agilliland** [comment]
cool :+1:

File: docs/users-guide/05-sharing-answers.md [hunk 1]
```diff
@@ -24,27 +24,55 @@ You can make as many dashboards as you want. Go nuts.
...
 Once you add a question to your dashboard, it’ll look something like this:
 
-![First Dashboard](images/FirstDashboard.png)
+![First Dashboard](images/dashboards/FirstDashboard.png)
 
 ### Arranging cards
 Each question on a dashboard is in its own card that you can move around or resize as you see fit; just click the edit icon that looks like a pencil in the top-right of the dashboard screen.  
 
-The lines around the card will change from solid to dotted, showing you that you’re in edit mode.  
+Once you're in edit mode you'll see a grid appear. You move and resize the cards in the dashboard to your liking and they'll snap to the grid.
```
Comment (mazameli) [inline docs/users-guide/05-sharing-answers.md]: > You move and resize the cards in the dashboard to your liking and they'll snap to the grid.

"You **can** move…"

File: docs/users-guide/05-sharing-answers.md [hunk 2]
```diff
@@ -24,27 +24,55 @@ You can make as many dashboards as you want. Go nuts.
...
-* To remove a card, click the trashcan icon on it.
 
 ## Deleting a dashboard
 Deleting a dashboard does not delete the individual saved questions on it — it just deletes the dashboard. Remember — dashboards are shared by everyone on your team, so think twice before you delete something that someone else might be using!
 
 To delete a dashboard, click the pencil-looking **Edit** icon in the top right of the dashboard, then click **Delete**.
 
+## Fullscreen dashboards
+
+After you've made your ideal dashboard you may want to put it on a TV or present it in some other visible space to help keep your team up to date through out the day.
```
Comment (mazameli) [inline docs/users-guide/05-sharing-answers.md]: "up-to-date" and "throughout"

File: docs/users-guide/05-sharing-answers.md [hunk 3]
```diff
@@ -24,27 +24,55 @@ You can make as many dashboards as you want. Go nuts.
...
+
+To enter fullscreen mode just click the fullscreen icon in the top right of the dashboard.
+
+Once you've entered fullscreen mode you can also switch the dashboard into "Night mode" to
```
Comment (mazameli) [inline docs/users-guide/05-sharing-answers.md]: There's a trailing sentence here.

File: docs/users-guide/05-sharing-answers.md (hunk 1 above)
Comment (kdoh) [inline docs/users-guide/05-sharing-answers.md]: it moves the cards around the dashboard

**Pull request merged**